.changelog/24198.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+resource/aws_s3_bucket_website_configuration: Add `routing_rules` parameter to be used in-place of `routing_rule` to support configurations with empty String values
```

Comment:
```suggestion
resource/aws_s3_bucket_website_configuration: Add `routing_rules` parameter to be used instead of `routing_rule` to support configurations with empty String values
```